packages/webdriverio/src/networkManager.ts

```diff
@@ -143,8 +147,27 @@ export class NetworkManager {
         request.error = log.errorText
     }
 
+    #findRootRequest (navigationId: string | null) {
+        const response = this.#requests.get(navigationId || UNKNOWN_NAVIGATION_ID)
+        if (response) {
+            return response
+        }
+
+        /**
+         * Chrome v127 and below does not support yet navigation ids, hence we have to
+         * just pick the first request object and assume it is the root request. This
+         * will break if the user operates on multiple windows at the same time.
+         *
+         * @see https://github.com/GoogleChromeLabs/chromium-bidi/issues/1054
+         */
+        const firstRequest = this.#requests.values().next().value
+        return this.#lastNetworkId
```

Comment:
will this not cause issues on other versions? just double-checking :)

Reply:
I will raise a PR next week to run these bidi tests also in Firefox. Right now they only run in Chrome where the navigation id is not yet available. I am not to what you are referring to when saying "other versions". The way it is implemented right now is broken (uses context id instead of navigation id).